modules/claims_api/config/schemas/v2/526.json

```diff
@@ -69,11 +69,13 @@
                 "properties": {
```

Comment:
Should we add a pattern for the email? Something like `EMAIL_REGEX = '/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/'`

Reply:
Added an email validation pattern and test for same.